dbms/src/Interpreters/Context.cpp

```diff
@@ -1850,6 +1855,23 @@ void Context::initializeSystemLogs()
     shared->system_logs.emplace(*global_context, getConfigRef());
 }
 
+void Context::createSystemLogs()
+{
+    auto lock = getLock();
+    if (shared->system_logs->query_log)
```

Comment:
But there is `struct SystemLogs` that encapsulate the set of available system logs.
It's no good to duplicate the list of logs here.